posthog/api/utils.py

```diff
@@ -23,19 +23,19 @@ def get_target_entity(request: request.Request) -> Entity:
         raise ValueError("An entity must be provided for target entity to be determined")
 
 
-def format_next_url(request: request.Request, offset: int, page_size: int):
+def format_next_url(request: request.Request, offset: int, page_size: int, previous: Optional[bool] = None):
```

Comment:
`Optional[bool]` doesn't really make sense - you want `previous = False` here instead.

Also the method name is now confusing. Is there a better way to name this function given it handles previous page as well?

Reply:
`format_paginate_url`?

Reply:
`format_paginated_url` sounds even better